addons/website_slides/models/slide_channel.py

```diff
@@ -47,6 +48,51 @@ class ChannelUsersRelation(models.Model):
         )
     ]
 
+    def _compute_next_slide_id(self):
+        query = """
+            WITH slides AS (
+                SELECT
+                    SS.sequence,
+                    SS.id AS slide_id,
+                    SS.channel_id AS channel_id,
+                    p.id AS partner_id
+                FROM
+                    res_partner p,slide_slide SS
+                WHERE
+                    ss.channel_id IN %s AND
+                    p.id IN %s AND
+                    ss.is_published = 't' AND
+                    ss.active = 't' AND
+                    NOT EXISTS(
+                        SELECT 1 FROM slide_slide_partner
+                        WHERE slide_id = SS.id
+                        AND partner_id = p.id
+                        AND completed = 't'
+                    )
+                ORDER BY SS.sequence
+            )
+            SELECT
+                next_slides.*
+            FROM
+                (
+                    SELECT channel_id, partner_id, min(sequence) AS sequence
+                    FROM slides
+                    GROUP BY channel_id, partner_id) min_slides
+            JOIN
+                slides next_slides
+            ON
+                next_slides.sequence = min_slides.sequence AND
+                next_slides.channel_id = min_slides.channel_id AND
+                next_slides.partner_id = min_slides.partner_id
+        """
+        self.env.cr.execute(query, [tuple(self.mapped('channel_id').ids), tuple(self.mapped('partner_id').ids)])
```

Comment:
nothing changed...

Reply:
Hello @std-odoo 

I am about to push the changes,
I have just answered the comment.

Thanks